tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+- name: Set login shell for user www-data
+  user: name=www-data shell=/bin/bash
+
+- name: Clone the opencraft repository
+  git: repo=https://github.com/open-craft/opencraft.git dest="{{ opencraft_root_dir }}" update=no
+  become_user: www-data
+
+- name: Install system dependencies
+  command: make install_system_dependencies chdir="{{ opencraft_root_dir }}"
+
+- name: Install virtualenv
+  pip: name=virtualenv executable=pip3
```

Comment:
We could make our Ansible code flexible enough to be used to provision the devstack (similar to how this is handled in edx-platform), but that would indeed be a separate project.  I'll take a note to create a task.